data/class/SC_Fpdf.php

```diff
@@ -311,6 +330,22 @@ private function setOrderData()
         }
 
         $this->FancyTable($this->label_cell, $arrOrder, $this->width_cell);
+
+        $this->SetLineWidth(.3);
+        $this->SetFont('SJIS', '', 6);
+
+        $this->Cell(0, 0, '', 0, 1, 'C', 0, '');
+        // 行頭近くの場合、表示崩れがあるためもう一個字下げする
+        if (270 <= $this->GetY()) {
+            $this->Cell(0, 0, '', 0, 1, 'C', 0, '');
+        }
+        $width = array_reduce($this->width_cell, function ($n, $w) {
+            return $n + $w;
+        });
+        $this->SetX(20);
+
+        $message = SC_Helper_TaxRule_Ex::getTaxDetail($arrTaxableTotal, $discount_total);
+        $this->MultiCell($width, 4, $message, 0, 'R', 0, '');
```

Comment:
```suggestion
        $this->MultiCell($width, 4, $message, 0, 'R', '');
```
とりあえず、SC_Fpdf.phpに関してはこれでテスト通ると思いますが、
e2e-tests/test/installer/installer.test.tsでCheck failureが出ているのは、よくわかりません。
確認お願いします。

Reply:
e2e-tests/test/installer/installer.test.ts はこのPRとは関係ないエラーだと思われるので大丈夫です。ありがとうございます！